nix/test-environments/example-setup-03.nix

```diff
@@ -73,6 +72,22 @@ in
               peg-tolerance-percentage = 0.000001; # 0.000001% tolerance assures that the price will not be pegged
             }
           ];
+          contracts = [
+            {
+              name = "AggregatedDataFeedStore";
+              address = upgradeableProxyADFSContractAddressInk;
+              creation-byte-code = null;
+              deployed-byte-code = null;
+              contract-version = 2;
+            }
+            {
+              name = "multicall";
+              address = "0xcA11bde05977b3631167028862bE2a173976CA11";
+              creation-byte-code = null;
+              deployed-byte-code = null;
+              contract-version = 3;
+            }
+          ];
```

Comment:
This seems fine

Reply:
Yet `multicall` probably is different for each network. Here we fork `ink-sepolia` but in the future we might change that. We might need to read if from a file in the future.